srcDLL/DllMain.cpp

```diff
@@ -98,13 +98,15 @@ void OnInit()
 
 	// Find VOL files from additional folders
 	for (std::size_t i = 0; i < moduleLoader->Count(); ++i) {
-		std::string moduleDirectory = moduleLoader->GetModuleDirectory(i);
+		const auto moduleDirectory = moduleLoader->GetModuleDirectory(i);
 		if (!moduleDirectory.empty()) {
-			LocateVolFiles(moduleLoader->GetModuleName(i));
+			LocateVolFiles(moduleDirectory);
 		}
 	}
-	LocateVolFiles("Addon");
-	LocateVolFiles(); //Searches root directory
+
+	const auto exeDirectory = fs::path(GetExeDirectory());
+	LocateVolFiles((exeDirectory / fs::path("Addon")).string());
```

Comment:
No need for the `fs::path` wrapper. A `std::string` or `const char*` will work just as well with `operator /`.

Reply:
Changing it to read:
```C++
LocateVolFiles(GetExeDirectory() / "Addon");
```

Produces: 
E0349	no operator "/" matches these operands	op2extDLL	\op2ext\srcDLL\DllMain.cpp	108

Perhaps the operator overload for path concatenation is mingw specific?

Reply:
Yes, correct. The line above originally read `fs::path(GetExeDirectory())`. That wrapping is enough. The `operator/` is overloaded to take both `operator/(fs::path, fs::path)` and `operator/(fs::path, std::string)`. It does not however take `operator/(std::string, std::string)`. As long as the first parameter is wrapped in `fs::path` it will work.

Reply:
PR#232 is addressing this.